### ISSUE TITLE
Any way to force keep it connected on Android?

### ISSUE THREAD

**Issue by elmar001** [wonjerry/flutter_wifi_connector#25]
Android 11, 12 makes it very hard for iOT devices. It is impossible to connect and configure them as Android detects that the connected WIFI  does not have internet connection. Flutter socket connect directly goes into catch exception block with timeout exception.

**Comment by kakopappa** [comment]
Hi, did you solve this problem?

**Comment by elmar001** [comment]
No, no solution. The problem is Android OS. Reported to Google but did not get any solution:
https://issuetracker.google.com/issues/238443303
